tests/test_scripts.py

```diff
@@ -2369,7 +2412,10 @@ def test_count_overlap_invalid_datafile():
     args = ['--ksize', '20', '--n_tables', '2', '--min-tablesize', '10000000',
             htfile + '.pt', htfile + '.pt', outfile]
     (status, out, err) = utils.runscript(script, args, in_dir, fail_ok=True)
-    assert "IOError" in err
+    if sys.version_info.major == 2:
+        assert "IOError" in err
+    else:
+        assert "OSError" in err
```

Comment:
`PyExc_IOError` will be converted to an `OSError` on Python 3.3+. (Different exceptions were merged into OSError, see https://docs.python.org/3/library/exceptions.html#OSError)